subsys/logging/log_backend_uart.c

```diff
@@ -69,11 +69,55 @@ static void dropped(const struct log_backend *const backend, u32_t cnt)
 	log_output_dropped_process(&log_output, cnt);
 }
 
+static void sync_msg(const struct log_backend *const backend,
```

Comment:
These functions are very similar in all backends. Could we make them common?

Reply:
i agree but let's do that as another PR.  Currently it is using global `log_output` variable. Reworking that will add changes unrelated to that PR.